nncf/quantization/algorithms/fast_bias_correction/onnx_backend.py

```diff
@@ -49,6 +54,10 @@ def tensor_processor(self) -> ONNXNNCFCollectorTensorProcessor:
     def target_point(target_type: TargetType, target_node_name: str, port_id: int) -> ONNXTargetPoint:
         return ONNXTargetPoint(target_type, target_node_name, port_id)
 
+    @staticmethod
+    def create_bias_insertion_command(node: NNCFNode) -> ONNXNullBiasInsertionCommand:
+        return ONNXNullBiasInsertionCommand(node)
```

Comment:
Does the ONNX model transformer support `ONNXNullBiasInsertionCommand` insertion command? What happens if such command will be feed to the ONNX model transformer?

Reply:
It's not support now. Currently, nothing will happen, just skipping in the `ONNXModelTransformer`.